<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{

	background-color : #ffffff;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="4">
     <widget class="QPushButton" name="Equals">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#00bfff;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="Button_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#7fffd4;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="All_Clear">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#ff0000;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="Button_0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#7fffd4;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}
</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="Button_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#7fffd4;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="Decimal_Point">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#00bfff;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Button_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#7fffd4;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="Percentage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#00ff00;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Button_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#7fffd4;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Button_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#7fffd4;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="Button_9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#7fffd4;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="Button_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#7fffd4;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="Division">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#00ff00;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="Clear">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#ff0000;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="Subtraction">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#00ff00;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="Button_8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#7fffd4;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="Multiplication">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#00ff00;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="Addition">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#00ff00;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="Button_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#7fffd4;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="Change_Sign">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#00ff00;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="Natural_Logarithm">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#c21e56;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>ln</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="6">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color : grey;
	border : 2px solid grey;
	color : #ffffff;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="Inverse">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#00bfff;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>1/X</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="SquareRoot">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#00bfff;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>√X</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="EulersNumber">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#c21e56;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>e</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="Sin">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#fc6600;
	border: 2px solid grey;
	padding: 5px;
	border-radius:10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="Cos">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#fc6600;
	border: 2px solid grey;
	padding: 5px;
	border-radius:10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="Tan">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#fc6600;
	border: 2px solid grey;
	padding: 5px;
	border-radius:10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="Power">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#00bfff;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>X^Y</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="Factorial">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#00bfff;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>X!</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="Logarithm">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#c21e56;
	border: 2px solid grey;
	padding: 5px;
	border-radius: 10px;
}

QPushButton : pressed {
	background-color:#00FFFF;
	border: 2px solid grey;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>log</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>474</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
